tests/generator/BGenTool.cs

```diff
@@ -88,54 +90,78 @@ string BuildArguments ()
 			}
 
 			if (!string.IsNullOrEmpty (targetFramework))
-				sb.Append (" --target-framework=").Append (targetFramework);
+				sb.Add ($"--target-framework={targetFramework}");
 
 			foreach (var ad in ApiDefinitions)
-				sb.Append (" --api=").Append (StringUtils.Quote (ad));
+				sb.Add ($"--api={ad}");
```

Comment:
are all those `StringUtils.Quote` removal intentional ? because even if we don't use them today we might need them inn future tests

Reply:
Yes, the code is now adding all the arguments (unquoted) to an array. If we're calling Main, we pass the array (without quoting). If we call an out-of-process bgen, we join each element of the array to a string, quoting when needed.